packages/constants/src/lib/constants/constants.ts

```diff
@@ -974,31 +1003,78 @@ export const PKP_CLIENT_SUPPORTED_CHAINS = ['eth', 'cosmos'];
 // ========== RLI Delegation ==========
 export const SIWE_DELEGATION_URI = 'lit:capability:delegation';
 
+const RELAY_URL_CAYENNE_TARGET = {
+  url: 'https://relayer-server-staging-cayenne.getlit.dev',
+};
 /**
  * @deprecated Will be removed in version 7.x. - Use RELAYER_URL_BY_NETWORK.Cayenne instead
  */
-export const RELAY_URL_CAYENNE =
-  'https://relayer-server-staging-cayenne.getlit.dev';
+export const RELAY_URL_CAYENNE = new Proxy(RELAY_URL_CAYENNE_TARGET, {
```

Comment:
I think we can just remove this as we already have this comment?
```
@deprecated Will be removed in version 7.x.
```

Reply:
Removed